### ISSUE TITLE
Default elastic user and password don't work.

### ISSUE THREAD

**Issue by shakai1** [deviantony/docker-elk#707]
I have just installed the docker-elk following the instructions provided.
Unfortunatly default user: elastic and password: changeme don't work.

**Comment by antoineco** [comment]
I'm not sure I understand how I'm supposed to help you with no information at all.

Quoting our [issue template](https://raw.githubusercontent.com/deviantony/docker-elk/main/.github/ISSUE_TEMPLATE/issue_report.md):

> In order for us to be able to reproduce the problem and identify the root cause
quickly, we kindly ask you to include *all* the information requested below in
your issue report. It saves us a lot of effort and allows us to provide you
with a solution with as little delay as possible.
>
> Issues submitted without the requested information **will be closed**.
> Thank you for your understanding.

I'll allow some time for you to edit your original issue description according to the template, and close the issue otherwise.

**Comment by shakai1** [comment]
I have a fresh installation of Ubuntu-20.04 LTS server.

I have updated the packages:
`sudo apt update ; sudo apt upgrade -y`

I have cloned the docker-elk
`git clone https://github.com/deviantony/docker`

I have installed docker-compose:
` sudo apt install docker-compose -y`

`docker-compose up`

The elastic GUI works http://localhost:5601/

Username: elastic 
password:changeme  

It doesnt work. Login or password incorrect.

**Comment by antoineco** [comment]
At least you could have had the courtesy to include the data I requested in your response, such as:

```console
$ docker-compose version
[OUTPUT HERE]
```

Anyway, I did my little research and the version of Docker Compose included with Ubuntu 20.04 ("universe" repository) is [1.25.0-1](https://packages.ubuntu.com/focal/docker-compose).

Which, as stated in the README, isn't supported:

> ## Requirements
>
>### Host setup
>
>* Docker Engine version **18.06.0** or newer
>* Docker Compose version **1.26.0** or newer (including Compose V2)

The reason why these older versions aren't supported by docker-elk was mentioned in a few issues already.
I think #698 is a good starting point.

Long story short: in Compose versions < v1.26, quotes aren't removed around the values inside `.env` files, so passwords end up being literally `'changeme'` (with quotes) instead of `changeme` (quotes removed).
It's a Compose bug. As a workaround, you could either upgrade, or remove the quotes from the `.env` file.

Closing as another duplicate of #696.

**Comment by shakai1** [comment]
$ docker version
Client:
 Version:           20.10.12
 API version:       1.41
 Go version:        go1.16.2
 Git commit:        20.10.12-0ubuntu2~20.04.1
 Built:             Wed Apr  6 02:14:38 2022
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

$ docker-compose version
docker-compose version 1.25.0, build unknown
docker-py version: 4.1.0
CPython version: 3.8.10
OpenSSL version: OpenSSL 1.1.1f  31 Mar 2020

k$ docker-compose logs
Traceback (most recent call last):
  File "/usr/bin/docker-compose", line 11, in <module>
    load_entry_point('docker-compose==1.25.0', 'console_scripts', 'docker-compose')()
  File "/usr/lib/python3/dist-packages/compose/cli/main.py", line 72, in main
    command()
  File "/usr/lib/python3/dist-packages/compose/cli/main.py", line 125, in perform_command
    project = project_from_options('.', options)
  File "/usr/lib/python3/dist-packages/compose/cli/command.py", line 53, in project_from_options
    return get_project(
  File "/usr/lib/python3/dist-packages/compose/cli/command.py", line 135, in get_project
    config_details = config.find(project_dir, config_path, environment, override_dir)
  File "/usr/lib/python3/dist-packages/compose/config/config.py", line 286, in find
    filenames = get_default_config_files(base_dir)
  File "/usr/lib/python3/dist-packages/compose/config/config.py", line 313, in get_default_config_files
    (candidates, path) = find_candidates_in_parent_dirs(SUPPORTED_FILENAMES, base_dir)
  File "/usr/lib/python3/dist-packages/compose/config/config.py", line 349, in find_candidates_in_parent_dirs
    if os.path.abspath(parent_dir) != os.path.abspath(path):
  File "/usr/lib/python3.8/posixpath.py", line 379, in abspath
    cwd = os.getcwd()
FileNotFoundError: [Errno 2] No such file or directory
Error in sys.excepthook:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/apport_python_hook.py", line 76, in apport_excepthook
    binary = os.path.realpath(os.path.join(os.getcwd(), sys.argv[0]))
FileNotFoundError: [Errno 2] No such file or directory

Original exception was:
Traceback (most recent call last):
  File "/usr/bin/docker-compose", line 11, in <module>
    load_entry_point('docker-compose==1.25.0', 'console_scripts', 'docker-compose')()
  File "/usr/lib/python3/dist-packages/compose/cli/main.py", line 72, in main
    command()
  File "/usr/lib/python3/dist-packages/compose/cli/main.py", line 125, in perform_command
    project = project_from_options('.', options)
  File "/usr/lib/python3/dist-packages/compose/cli/command.py", line 53, in project_from_options
    return get_project(
  File "/usr/lib/python3/dist-packages/compose/cli/command.py", line 135, in get_project
    config_details = config.find(project_dir, config_path, environment, override_dir)
  File "/usr/lib/python3/dist-packages/compose/config/config.py", line 286, in find
    filenames = get_default_config_files(base_dir)
  File "/usr/lib/python3/dist-packages/compose/config/config.py", line 313, in get_default_config_files
    (candidates, path) = find_candidates_in_parent_dirs(SUPPORTED_FILENAMES, base_dir)
  File "/usr/lib/python3/dist-packages/compose/config/config.py", line 349, in find_candidates_in_parent_dirs
    if os.path.abspath(parent_dir) != os.path.abspath(path):
  File "/usr/lib/python3.8/posixpath.py", line 379, in abspath
    cwd = os.getcwd()
FileNotFoundError: [Errno 2] No such file or directory
